mbed/mbed.py

```diff
@@ -1294,6 +1295,14 @@ def get_tools_dir(self):
 
         return self._find_file_paths(paths, 'make.py')
 
+    def get_tools_command(self, script):
+        if os.name != 'nt' \
```

Comment:
Could you please reverse the logic here, so standalone is the special case and the default is the .py case?

Reply:
Bump @janjongboom